packages/flutter/lib/src/material/chip.dart

```diff
@@ -1220,10 +1250,11 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
   Animation<double> enableAnimation;
   Animation<double> selectionFade;
 
-  static final Animatable<double> pressedShadowTween = Tween<double>(
-    begin: 0.0,
-    end: _kPressElevation,
-  );
+
+//  static final Animatable<double> pressedShadowTween = Tween<double>(
```

Comment:
What was this used for?

Reply:
rip, that was there before I started working on it.